controllers/operands/kubevirt.go

```diff
@@ -37,6 +37,8 @@ const (
 const (
 	DefaultAMD64OVMFPath         = "/usr/share/OVMF"
 	DefaultAMD64EmulatedMachines = "q35*,pc-q35*"
+	DefaultARM64OVMFPath         = "/usr/share/OVMF"
+	DefaultARM64EmulatedMachines = "q35*,pc-q35*"
```

Comment:
I don't have a clue if these values I have assumed are accurate? Can someone validate so that I can open the PR for review?

@tiraboschi @nunnatsa

Reply:
according to this:
https://people.redhat.com/~cohuck/2022/01/05/qemu-machine-types.html
the QEMU machine-type for ARM should be `virt`.
but it would be better to confirm with virt team. @kbidarkar do you know?

Reply:
@acardace  could you please confirm arm64 machinetypes?

Reply:
`qc35` is only available for x86 so I'd avoid setting that. The equivalent of `qc35` for arm is `virt` like @orenc1 mentioned.

Reply:
@orenc1 @acardace thanks for your help. 👍🏽